include/drivers/gps.h

```diff
@@ -95,6 +119,12 @@ struct gps_config {
 	/* Power mode, @ref enum gps_power_mode. */
 	enum gps_power_mode power_mode;
 
+	/* Accuracy threshold for producing fix, @ref enum gps_use_case. */
+	enum gps_use_case use_case;
```

Comment:
It looks like the first part of the above description should be updated.

Reply:
Indeed, fixed now